nextjs/pages/index.tsx

```diff
@@ -6,42 +6,54 @@ import ToggleButton from "./components/ToggleButton";
 
 export default function LearningPage() {
 
-    const buttonCSS = "bg-blue-500 hover:bg-green-700 text-white font-bold py-2 px-4 rounded";
-
-    const [showWebcam, setShowWebcam] = useState(false)
     const [showSignTutorial, setShowSignTutorial] = useState(false);
+    const [[currentSign, URL], setCurrentSign] = useState(getNextSign());
+
+    const [showWebcam, setShowWebcam] = useState(true);
+    const [isCapturing, setIsCapturing] = React.useState(false);
+    const [blobURL, setBlobURL] = React.useState("");
+    const [shouldAnalyse, setShouldAnalyse] = useState(false);
 
     const closeSignTutorial = () => setShowSignTutorial(false);
     const displaySignTutorial = () => setShowSignTutorial(true);
 
-    const [[currentSign, URL], setCurrentSign] = useState(getNextSign());
+    const buttonCSS = "bg-blue-500 hover:bg-green-700 text-white font-bold py-2 px-4 rounded";
 
     /* Should analyse toggle switch state*/
-    const [shouldAnalyse, setShouldAnalyse] = useState(false);
 
     return (
         <>
             {showSignTutorial && <SignTutorial signName={currentSign!} url={URL!} closeModal={closeSignTutorial} />}
-
             <div className="p-6 mx-auto bg-slate-200 mt-10 rounded-xl shadow-lg flex flex-col w-fit gap-8">
                 <h1 className="text-center text-3xl font-semibold">ASL recognizer: {currentSign === undefined ? "Finished!" : currentSign}</h1>
 
                 <div className="self-center">
-                    {showWebcam ? <WebcamCapture shouldAnalyse={shouldAnalyse} signLabel={currentSign!} /> : <ReactPlayer url="sign_videos/signvid.webm" controls={true} />}
+                    {/* TODO React player displays "You haven't recorded a video yet" */}
+                    {/* TODO React player can play recorded video */}
+                    {showWebcam ? 
+                        <WebcamCapture isCapturing={isCapturing} setIsCapturing={setIsCapturing} hideWebcam={() => setShowWebcam(false)} shouldAnalyse={shouldAnalyse} signLabel={currentSign!} setBlobURL = {setBlobURL} /> 
+                        : 
+                        <ReactPlayer url={blobURL} controls={true} />}
```

Comment:
```suggestion
                    {showWebcam && <WebcamCapture isCapturing={isCapturing} setIsCapturing={setIsCapturing} hideWebcam={() => setShowWebcam(false)} shouldAnalyse={shouldAnalyse} signLabel={currentSign!} setBlobURL = {setBlobURL} />}
                        {!showWebcam && <ReactPlayer url={blobURL} controls={true} />}
```

Reply:
no

Reply:
er sådan dybt uenig her

Reply:
så må du diskutere din pointe, for jeg er giga no på det der

Reply:
Synes det gør det langt sværre at læse hvad der foregår når at conditonal er 3-4 linjer over. Desuden kan jeg ikke lide den flydende kolon 

Kan se at der er nogle der gør det sådan her
![image](https://user-images.githubusercontent.com/56830573/167792234-1e054f79-304c-45ee-9ec2-00c8ed42dde1.png)
Og det kan jeg i det mindste acceptere. Men synes stadig mit foreslag er mere readable.